pkgs/top-level/release-cuda.nix

```diff
@@ -69,98 +72,147 @@ let
-        grad-cam = linux;
-        jaxlib = linux;
-        jax = linux;
-        Keras = linux;
-        kornia = linux;
-        mmcv = linux;
-        mxnet = linux;
-        numpy = linux; # Only affected by MKL?
-        onnx = linux;
-        triton = linux;
-        openai-whisper = linux;
-        opencv4 = linux;
-        opensfm = linux;
-        pycuda = linux;
-        pymc = linux;
-        pyrealsense2WithCuda = linux;
-        pytorch-lightning = linux;
-        pytorch = linux;
-        scikitimage = linux;
-        scikit-learn = linux; # Only affected by MKL?
-        scipy = linux; # Only affected by MKL?
-        spacy-transformers = linux;
-        tensorflow = linux;
-        tensorflow-probability = linux;
-        tesserocr = linux;
-        Theano = linux;
-        tiny-cuda-nn = linux;
-        torchaudio = linux;
-        torch = linux;
-        torchvision = linux;
-        transformers = linux;
-        ttstokenizer = linux;
-        vidstab = linux;
-        vllm = linux;
-      };
-    }
-    // (lib.genAttrs packageSets evalPackageSet);
+      faster-whisper = linux;
+      flax = linux;
+      gpt-2-simple = linux;
+      grad-cam = linux;
+      jaxlib = linux;
+      jax = linux;
+      keras = linux;
+      kornia = linux;
+      mmcv = linux;
+      mxnet = linux;
+      numpy = linux; # Only affected by MKL?
+      onnx = linux;
+      triton = linux;
+      openai-whisper = linux;
+      opencv4 = linux;
+      opensfm = linux;
+      pycuda = linux;
+      pymc = linux;
+      pyrealsense2WithCuda = linux;
+      pytorch-lightning = linux;
+      scikit-image = linux;
+      scikit-learn = linux; # Only affected by MKL?
+      scipy = linux; # Only affected by MKL?
+      spacy-transformers = linux;
+      tensorflow = linux;
+      tensorflow-probability = linux;
+      tesserocr = linux;
+      tiny-cuda-nn = linux;
+      torchaudio = linux;
+      torch = linux;
+      torchvision = linux;
+      transformers = linux;
+      ttstokenizer = linux;
+      vidstab = linux;
+      vllm = linux;
+    };
+  };
+
+  evalPackagePlatforms = lib.recursiveUpdate evalPackageSetPlatforms evalIndividualPackagePlatforms;
+
+  # Packages that match the following predicates won't be evaluated on any platforms
+  # Each predicate takes a human-readable, dot-delimited attribute path and the derivation itself
+  filterPackagePredicates =
+    let
+      matchCudaPackageByName = name: path: null != builtins.match "cudaPackages[^\.]*\.${name}" path;
+      knownBrokenConditions =
+        conds: drv: builtins.any (cond: drv.brokenConditions.${cond} or false) conds;
+    in
+    [
+      # TensorRT currently can't be built on hydra/nix-community builders due to requiring the user
+      # to explicitly agree to the NVIDIA TensorRT License (and a developer account on developer.nvidia.com)
+      # See https://docs.nvidia.com/deeplearning/tensorrt/latest/reference/sla.html
+      (drv: matchCudaPackageByName "tensorrt[^\.]*")
```

Comment:
Another option could be to filter `requireFile` derivations instead

Reply:
My intention with this PR was to be maximally specific and only filter the packages that we know for sure to be "false positive" eval failures. That way, we won't accidentally silence any future eval failures. And if there are ever anymore "false positives", adding them to this list should be trivial.

Reply:
You mean in case TensorRT changes its license one day?

Also, is `isRequireFile` a thing? I couldn't find anything like that by grepping in nixpkgs.

AFAIK, `requireFile` is just a thin wrapper around a `stdenvNoCC.mkDerivation` FOD that prints an error if you try to build it. I am not sure if there is any way to dynamically detect it.

Reply:
> Also, is isRequireFile a thing? I couldn't find anything like that by grepping in nixpkgs.

No, sorry, I made that up for brevity

Reply:
I don't think it's possible to implement something like `isRequireFile` in a robust way, so I am marking this as resolved for now (unless you have any suggestions).